### PR TITLE
PP-7711 Add ability to create AGENT_INITIATED_MOTO products

### PR DESCRIPTION
**Pull request by alexbishop1**
Add a page in toolbox to allow us to create an `AGENT_INITIATED_MOTO` product for a gateway account (along with an API key for it).

image

### CODE REVIEW

File: src/web/modules/gateway_accounts/detail.njk [hunk 1]
```diff
@@ -204,6 +204,11 @@
         href: "/gateway_accounts/" + gatewayAccountId + "/email_branding"
       })
       }}
+      {{ govukButton({
```
Comment (sfount) [inline src/web/modules/gateway_accounts/detail.njk:207]: I wonder if in the longer term it's worth moving this under the same category as the "Enable MOTO payments" section beneath this. Potentially making this less prominent in the context of that being enable/ disabled would help avoid confusion.

This seems to be a pattern for now so nothing this PR has to decide.

File: src/web/modules/gateway_accounts/gateway_accounts.http.ts [hunk 2]
```diff
@@ -21,6 +22,9 @@ import HTTPSProxyAgent from 'https-proxy-agent'
 import { format } from './csv'
 import { formatWithAdminEmails } from './csv_with_admin_emails'
 import { createCsvWithAdminEmailsData, createCsvData } from './csv_data'
+import CreateAgentInitiatedMotoProductFormRequest from './CreateAgentInitiatedMotoProductFormRequest'
+import { formatErrorsForTemplate,  } from '../common/validationErrorFormat'
```
Comment (sfount) [inline src/web/modules/gateway_accounts/gateway_accounts.http.ts:26]:
```suggestion
import { formatErrorsForTemplate } from '../common/validationErrorFormat'
```

**Review by sfount — DISMISSED** [review]
Code looks good, I haven't tried this out locally but it follows existing patterns and seems sensible. 

As mentioned it looks like as gateway account operations grow we might want to do a quick rethink about how we organise them on the page, not needed for this change.

File: src/web/modules/gateway_accounts/detail.njk (hunk 1 above)
Comment (alexbishop1) [reply src/web/modules/gateway_accounts/detail.njk:207]: Yeah, I think the navigation needs a bit of a rethink (you have to go to 3 different places to configure everything necessary for agent-initiated MOTO payments to work!) but I just followed the existing conventions for now.

File: src/web/modules/gateway_accounts/gateway_accounts.http.ts (hunk 2 above)
Comment (alexbishop1) [reply src/web/modules/gateway_accounts/gateway_accounts.http.ts:26]: Whoops!

**Review by sfount — APPROVED** [review]

**Pull request merged**